app/mailers/digest_mailer.rb

```diff
@@ -1,17 +1,49 @@
-class DigestMailer < ActionMailer::Base
-  default from: ENV["MAIL_NOTIFICATION"]
+class DigestMailer < ApplicationMailer
 
-  # Subject can be set in your I18n file at config/locales/en.yml
-  # with the following lookup:
-  #
-  # en.digest_mailer.daily_digest.subject
+  def daily_digest(history, user_id)
+    # @discussions = history.discussions
+    # @tutorials = history.tutorials
+    # @libraries = history.libraries
+    # @new_users_count = User.count - history.users_count
+    # @comment = Comment.find comment_id
+    @user = User.find user_id
 
-  def daily_digest(history, user)
-    @discussions = history.discussions
-    @tutorials = history.tutorials
-    @libraries = history.libraries
-    @new_users_count = User.count - history.users_count
-    @user = user
-    mail to: user.email, subject: 'RubyGameDev.com Daily Digest'
+    attachments.inline['header_daily.png'] =
+      File.read("#{Rails.root}/app/assets/images/header_daily.png")
+    attachments.inline['footer.png'] =
+      File.read("#{Rails.root}/app/assets/images/footer.png")
+    mail(to: @user.email, subject: 'RubyGameDev.com Daily Digest')
   end
-end
+
+  def weekly_digest(history, user_id)
```

Comment:
Unused method argument - `history`. If it's necessary, use `_` or `_history` as an argument name to indicate that it won't be used.